### DISCUSSION TITLE
Why does `aws_stepfunctions_tasks` only support a small subsets actions?

### DISCUSSION THREAD

**Discussion by JonathanWoollett-Light** [aws/aws-cdk#29120] [Q&A]
E.g. from https://docs.aws.amazon.com/codebuild/latest/APIReference/API_Operations.html only https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_stepfunctions_tasks.CodeBuildStartBuild.html is supported for [codebuild](https://aws.amazon.com/codebuild/). This makes integration significantly more awkward.

Are the wrappers not auto-generated? I'm curious why the support is quite narrow here.

**Comment by gshpychka** [comment]
Nope, they're not auto-generated. Here's the PR that added it: https://github.com/aws/aws-cdk/pull/9757
